runtime/src/assets/stylesheets/shared/_custom-variables.less

```diff
@@ -0,0 +1,16 @@
+@import (reference) './../../../../node_modules/patternfly/node_modules/bootstrap/less/variables';
+@import (reference) './../../../../node_modules/patternfly/src/less/variables';
+
+//custom variables
+@width100: 100%;
+@em-base-font-size: 12;
+@rem-base-font-size: 10;
+@pf-icon-2x: 24px;
+@pf-icon-3x: 36px;
+@pf-icon-4x: 48px;
+@textLeft: left;
+@textRight: right;
+@textCenter: center;
```

Comment:
remove and replace with `text-align: center;` across files, as this will never not be `text-align: center`, thus not qualifying as a variable under the code guidelines.